### ISSUE TITLE
CT Redesign Search: After clicking "See more..." focus should be on the revealed option

### ISSUE THREAD

**Issue by delli-gatti-michael** [department-of-veterans-affairs/va.gov-team#29265]
After conducting a search in the redesigned Comparison Tool, the user has the ability to filter by clicking checkboxes to exclude specific school types. Two of these types are hidden by default and can be revealed by clicking a "See more..." button. If a user is using a screen reader combined with a keyboard and selects the "See more..." button, the focus should be on the first option that is revealed. 

### Steps to reproduce
1. With a screen reader running, navigate to https://staging.va.gov/gi-bill-comparison-tool-sandbox/
2. Execute any search which returns multiple results (ex. "college" name search)
3. Tab to the "See more..." checkbox under the "Exclude these school types:" checkbox group
4. Press enter

![Screen Shot 2021-08-25 at 4 45 37 PM](https://user-images.githubusercontent.com/45575003/130863960-1fdccd7e-0e6c-4cef-9786-f529316a9112.png)

### Expected Result
Focus in now on the newly revealed "Flight" checkbox where the user can resume navigating the page
![Screen Shot 2021-08-25 at 4 58 49 PM](https://user-images.githubusercontent.com/45575003/130863839-3c287c97-698e-4a44-a561-592bd820bdad.png)

### Actual Result
Focus is lost

**Comment by zurbergram** [comment]
https://github.com/department-of-veterans-affairs/vets-website/pull/18537

**Comment by delli-gatti-michael** [comment]
local testing complete

**Comment by delli-gatti-michael** [comment]
Testing complete
